packages/lambda_streams/lambda_streams.0.1.2/opam

```diff
@@ -32,10 +31,9 @@ build: [
 ]
 dev-repo: "git+https://github.com/Risto-Stevcev/lambda-streams.git"
 url {
-  src:
-    "https://github.com/Risto-Stevcev/lambda-streams/archive/0.1.2.tar.gz"
+  src: "https://github.com/Risto-Stevcev/lambda-streams/archive/0.1.2.tar.gz"
   checksum: [
-    "sha256=a69205cd63d3be360f0a18a0ae287d8596b88a00405be821a4f6e6a809c44556"
+    "md5=8c1ce04ee769b56434696bd57aee1a5b"
```

Comment:
The previous hash is still correct. Why change this? I don't understand why this file has to change at all

Reply:
I didn't change it manually. I originally had trouble using `opam publish` because of a `git fetch` error, so I did a manual PR before. I managed to fix it by following what people said in the issues -- this one's coming directly from the `opam publish` command